management/src/test/java/org/ehcache/management/providers/statistics/HitCountTest.java

```diff
@@ -100,7 +101,14 @@ public void test() throws InterruptedException, IOException {
       registryConfiguration.addConfiguration(new EhcacheStatisticsProviderConfiguration(1,TimeUnit.MINUTES,100,1,TimeUnit.MILLISECONDS,10,TimeUnit.MINUTES));
       ManagementRegistryService managementRegistry = new DefaultManagementRegistryService(registryConfiguration);
 
-      CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class, resources).build();
+      CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class, resources)
+        .withEvictionAdvisor(new EvictionAdvisor<Long, String>() {
+          @Override
+          public boolean adviseAgainstEviction(Long key, String value) {
+            return key.equals(2L);
```

Comment:
@ljacomet : I would be interested in knowing why trying to prevent the eviction of only `key.equals(2L);` ? 
Perhaps we can add a little comment ? Just to be sure we follow the same pattern and understand the reason behind for future tests. Thanks :-)

CC @gibsong : FYI.